controller_manager/include/controller_manager/controller_manager.hpp

```diff
@@ -80,10 +80,14 @@ class ControllerManager : public rclcpp::Node
     const std::string & namespace_ = "",
     const rclcpp::NodeOptions & options = get_cm_node_options());
 
-  CONTROLLER_MANAGER_PUBLIC
```

Comment:
Shouldn't those be present on the Humble branch?

Reply:
You are right! I added them in the recent commit.

Reply:
Thank you! :)